src/main/java/org/springframework/retry/backoff/ExponentialBackOffPolicy.java

```diff
@@ -49,10 +49,10 @@ public class ExponentialBackOffPolicy implements SleepingBackOffPolicy<Exponenti
 	protected final Log logger = LogFactory.getLog(this.getClass());
 
 	/**
-	 * The default 'initialInterval' value - 100 millisecs. Coupled with the default
+	 * The default 'initialInterval' value - 1000 millisecs. Coupled with the default
 	 * 'multiplier' value this gives a useful initial spread of pauses for 1-5 retries.
 	 */
-	public static final long DEFAULT_INITIAL_INTERVAL = 100L;
+	public static final long DEFAULT_INITIAL_INTERVAL = 1000L;
```

Comment:
This is some kind of breaking change for the current point release, but I indeed see the reasoning behind consistency.
Plus it looks like the `@Backoff` annotation is always contributing exactly that `1000`  value.
So, we really can bite a bullet and let this pass through to the next release in a couple weeks.